containers/server-systemd-services/uyuni-server-services.config

```diff
@@ -38,3 +38,4 @@ SCC_USER=
 SCC_PASS=
 REPORT_DB_HOST=uyuni-server
 UYUNI_FQDN=uyuni-server
+TZ=Etc/UTC
```

Comment:
can't you just use `local` as a value? For `podman` I believe it does the same thing than what your script is doing. We can't use it in the helm chart, but for `podman` that would make the setup easier.

Reply:
unfortunately not, `local` does not work for TZ